src/main/java/htsjdk/samtools/cram/ref/EnaRefService.java

```diff
@@ -75,6 +77,12 @@
                     case HTTP_CONNECTION_TIMEOUT:
                     case HTTP_INTERNAL_SEVER_PROBLEM:
                         break;
+                    case HTTP_MOVED_PERMANENTLY:
+                        log.error("It seems that the base URL for the ENA service has changed permanently. Got error:" + code +
```

Comment:
It seems that this failure is recoverable as the response header/body contains the new URL... Perhaps to much work considering that these changes should not happen that often.

Reply:
I don't want to automatically follow a random URL that was given....sounds like not very secure behaviour